docs/api.md

```diff
@@ -292,6 +292,11 @@ content-server page.
     signed in email address will be used as the default.
   - If `action` is `force_auth`, the user is unable to modify the email
     address and is unable to sign up if the address is not registered.
+- `keys`: Optional. Boolean setting, set this if relier will need derive relier keys during the signup / signin flow.
+- `verification_redirect`: Optional. This option adds a "Proceed" button into the "Account Ready" view. See options for details.
```

Comment:
This is difficult to document too. A "Proceed" button is only displayed if the user verifies in a 2nd browser. If the user verifies in the same browser, they are automatically redirected w/o user interaction.